extensions/amp-next-page/0.1/amp-next-page.js

```diff
@@ -57,22 +61,36 @@ export class AmpNextPage extends AMP.BaseElement {
     }
 
     const {element} = this;
-
     element.classList.add('i-amphtml-next-page');
 
-    // TODO(peterjosling): Read config from another source.
-
-    const scriptElements = childElementsByTag(element, 'SCRIPT');
-    user().assert(scriptElements.length == 1,
-        `${TAG} should contain only one <script> child.`);
-    const scriptElement = scriptElements[0];
-    user().assert(isJsonScriptTag(scriptElement),
-        `${TAG} config should ` +
-            'be inside a <script> tag with type="application/json"');
-    const configJson = tryParseJson(scriptElement.textContent, error => {
-      user().error(TAG, 'failed to parse config', error);
-    });
+    const src = element.getAttribute('src');
+    if (src) {
```

Comment:
`assertHttpsUrl()` needed for value of `src`

Reply:
`batchFetchJsonFor()` calls `assertHttpsUrl()` (I used `amp-list` as an example)